Library/Formula/syncthing-inotify.rb

```diff
@@ -0,0 +1,63 @@
+class SyncthingInotify < Formula
+  desc "File watcher intended for use with Syncthing"
+  homepage "https://github.com/syncthing/syncthing-inotify"
+  url "https://github.com/syncthing/syncthing-inotify.git",
+    :tag => "v0.6.5", :revision => "2b290b929e18e7cd1cfea11240e4fa870d6638d5"
```

Comment:
when I change by this:
```ruby
  url "https://github.com/syncthing/syncthing-inotify/archive/v0.6.5.tar.gz"
  version "0.6.5"
  sha256 "430297896bb05396268fd29cc555eba6542b42263489784c9843f4daf625ac5c"
```
I get an error:
```
==> Downloading https://github.com/syncthing/syncthing-inotify/archive/v0.6.5.tar.gz
==> Downloading from https://codeload.github.com/syncthing/syncthing-inotify/tar.gz/v0.6.
######################################################################## 100,0%
Error: File exists - /Library/Caches/Homebrew/syncthing-inotify-0.6.5.tar.gz
```

It seems it download multiple times the same tarball...